src/assets/specs/command-api.yaml

```diff
@@ -112,6 +112,58 @@ paths:
             "text/plain":
               schema:
                 type: string
+
+  /v0/diagnostic_categories:
```

Comment:
Do we need separate endpoints here?

My understand is that we basically need three things:
1. A thing to list all diagnostics (and their categories), possibly in a fixed order.
2. A thing to run (possibly a subset of) the diagnostics, returning results which can include more detailed textual descriptions.
3. Possibly a thing to configure diagnostics (mute some of them, etc.)

Given that, have a separate endpoint for categories and ids seems unnecessary?

Reply:
Right now I'm thinking about the needs of the UI just to get started. If the UI is going to display diags by category first, it would be helpful to get the categories.

And I rules out an endpoint to get all the diagnostics in one call because we might end up with a lot of them and then we have pagination issues.

This might change later, but for now I think it solves the current problem.

Reply:
While the UI will be group by categories, it still needs all the diagnostics in one go (since it's not going to render it as collapsed categories, and only fetch the full list on expanding), right?

I think designing for pagination is overkill (because the _user_ still needs to be able to look at the list), but we'll see how the UI ends up looking before changing this I guess.